package.json

```diff
@@ -61,7 +62,7 @@
     "react-navigation": "3.9.1",
     "react-redux": "7.0.3",
     "realm": "2.28.1",
-    "realm-react-redux": "github:enahum/realm-react-redux#9e510a92e8c282a5f8a8ee912548831d37528692",
+    "realm-react-redux": "github:enahum/realm-react-redux#7f54c089fff81d578d1bf8a46d9230a5c20ab40e",
```

Comment:
When do you plan to move this package into our org?  Is it when this feature branch is merged into master?